src/std/main.ab

```diff
@@ -93,7 +93,7 @@ pub fun len(value): Num {
 }
 
 #[allow_absurd_cast]
-pub fun parse(text: Text): Num {
+pub fun parse(text: Text): Num ? {
```

Comment:
```suggestion
pub fun parse(text: Text): Num? {
```

Can we remove the whitespace in all of the instances? I thin that in the end we will create just a „failable data type” that basically has to be handled

Reply:
Sure, that will have to handled in the type parser instead.